README.rst

```diff
@@ -15,6 +15,8 @@ Introduction
 
 CircuitPython pixel graphics drawing library.
 
+Important: This library will likely be deprecated soon. If you are looking to use a specific screen, search the product name (i.e. ssd1322 or ili9341) `here <https://github.com/adafruit/>`_, or find it in the `circuitpython bundle <https://github.com/adafruit/Adafruit_CircuitPython_Bundle>`_.
```

Comment:
Just a nitpick here, but let's change screen to display. Also, with your "here" link, let's spell out the actual address in case somebody doesn't want to click it. So instead of:
```
search the product name (i.e. ssd1322 or ili9341) `here <https://github.com/adafruit/>`_
```
change it to
```
search the product name (i.e. ssd1322 or ili9341) at `https://github.com/adafruit/ <https://github.com/adafruit/>`_
```